__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,18 @@
+const Itinerary = require('../src/Itinerary.js');
+const Port = require('../src/Port.js');
+
+describe('Itinerary', () => {
+    it('can be instantiated', () => {
+        expect(new Itinerary()).toBeInstanceOf(Object);
+    });
+
+    it('has ports', () => {
+        const portsmouth = jest.fn();
+        const ilfracombe = jest.fn();
+        
+        const itinerary = new Itinerary([portsmouth, ilfracombe]);
+ 
+        expect(itinerary.ports).toEqual([portsmouth, ilfracombe]);
+ });
```

Comment:
These closing brackets should be on the same line where the it blocks starts on line 9